llvm/lib/Target/ARM/ARMAsmPrinter.cpp

```diff
@@ -680,14 +679,14 @@ void ARMAsmPrinter::emitAttributes() {
   if (isPositionIndependent()) {
     ATS.emitAttribute(ARMBuildAttrs::ABI_PCS_RW_data,
                       ARMBuildAttrs::AddressRWPCRel);
-  } else if (STI.isRWPI()) {
+  } else if (getTM().isRWPI()) {
```

Comment:
Would this get set if the user had not put command-line flags on the link step of their LTO.
```
clang -flto -frwpi input.c
clang input.o (other linker driver flags but no c-flags like -frwpi)
```

Although looking at the existing `ARMSubtarget::isROPI()` it looks like that may not have worked beforehand.